hammett/core/__init__.py

```diff
@@ -1,3 +1,4 @@
+from collections.abc import Iterable
```

Comment:
Move this import under TYPE_CHECKING and wrap `Iterable[type['Screen']]` in single quotes: `'Iterable[type[Screen]]'`